app/controllers/idv/forgot_password_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Idv
+  class ForgotPasswordController < ApplicationController
+    include IdvSession
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_needed
+
+    def new
+      analytics.track_event(Analytics::IDV_FORGOT_PASSWORD)
+      @presenter = ForgotPasswordPresenter.new(view_context: view_context)
+    end
+
+    def update
+      analytics.track_event(Analytics::IDV_FORGOT_PASSWORD_CONFIRMED)
+      request_id = sp_session[:request_id]
+      email = current_user.email
+      reset_password(email, request_id)
+    end
+
+    private
+
+    def reset_password(email, request_id)
+      sign_out
+      RequestPasswordReset.new(email, request_id).perform
+      # The user/email is always found so...
```

Comment:
Did you mean to add more after the ellipses here?

Reply:
It was implying that we didn't need to check the return value of the line above...

Reply:
Okay, I just didn't know if you meant to come back and finish it. Thoughts on fleshing it out more so future travelers don't have to run into the old password reset code for it to make sense?

Reply:
Would it be clearer if I just removed the comment?

Reply:
If you want. I think this is an appropriate place for a comment though